### PR TITLE
Change fader levels

### PR DESCRIPTION
**Pull request by mstratford**
Completely untested guesswork, brought to you from a garden chair.

### CODE REVIEW

**Comment by mstratford** [comment]
Not ready yet.

**Comment by mstratford** [comment]
Will need testing :)

File: src/mixer/state.ts [hunk 1]
```diff
@@ -568,6 +568,11 @@ export const setVolume = (
       }
     })
     .on("complete", () => {
+
+      if (level === 0) {
+        volume = -100; // Allow a smooth fade to near silence, then kill it dead.
```
Comment (markspolakovs) [inline src/mixer/state.ts:573]: You need the converse of this when fading from zero, otherwise we'll have the same problem that we had earlier.

**Comment by markspolakovs** [comment]
@mstratford is this still relevant?

**Comment by mstratford** [comment]
> @mstratford is this still relevant?

Still a problem, but this is likely not the solution.

**Pull request closed without merging**